lib/kvdb/kvdb_meta.c

```diff
@@ -548,6 +549,55 @@ kvdb_meta_usage(const char *kvdb_home, uint64_t *const size)
     return err;
 }
 
+merr_t
+kvdb_meta_version_check(const struct kvdb_meta *const meta)
+{
+    INVARIANT(meta);
+
+    if (meta->km_version == KVDB_META_VERSION && meta->km_omf_version == GLOBAL_OMF_VERSION)
+        return 0; /* Nothing to do */
+
+    if (meta->km_omf_version > GLOBAL_OMF_VERSION) {
```

Comment:
I think this might already be checked in the deserailize func, but you may have to check. Otherwise it should be added there.

Reply:
Added, thnx!